lib/modules/manager/npm/artifacts.ts

```diff
@@ -63,6 +67,11 @@ export async function updateArtifacts({
 
   const execOptions: ExecOptions = {
     cwdFile: packageFileName,
+    extraEnv: {
+      // need to align with lib/modules/manager/npm/post-update/pnpm.ts
```

Comment:
I recommend we reference a single definition in both location to guarantee this, instead of hoping this comment will magically fix it for us.